ddtrace/ddtrace.go

```diff
@@ -94,6 +98,17 @@ type SpanContext interface {
 	ForeachBaggageItem(handler func(k, v string) bool)
 }
 
+// https://github.com/open-telemetry/opentelemetry-go/blob/v1.21.0/trace/trace.go#L412
+//go:generate msgp
+type SpanLink struct {
+	TraceID     uint64            `msg:"trace_id"`                // Required. The low 64 bits of a referenced trace id
+	TraceIDHigh uint64            `msg:"trace_id_high,omitempty"` // Optional. The high 64 bits of a referenced trace id.
+	SpanID      uint64            `msg:"span_id"`                 // Required.
+	Attributes  map[string]string `msg:"attributes,omitempty"`    // Optional. Simple mapping of keys to string values.
+	Tracestate  string            `msg:"tracestate,omitempty"`    // Optional. W3C tracestate.
+	Flags       uint32            `msg:"flags,omitempty"`         // Optional. W3C trace flags. If set, the high bit (bit 31) must be set.
```

Comment:
Please put the docs in a comment above the field (see others for an example)

Reply:
(Unresolving since it looks like this didn't happen yet)

Reply:
ahh sorry I missed this. I was following this convention: https://github.com/DataDog/dd-trace-go/blob/main/ddtrace/tracer/span.go#L64. I now realize those span comments are never exported 😵‍💫. I will fix this in the next commit